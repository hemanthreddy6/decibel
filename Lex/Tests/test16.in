audio clip = audio(SAW,1.0,440);
clip = clip[:5];
play clip^4;
