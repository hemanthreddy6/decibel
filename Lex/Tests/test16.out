AUDIO
IDENTIFIER
=
AUDIO
(
SAW
,
FLOAT
,
NUMBER
)
;
IDENTIFIER
=
IDENTIFIER
[
:
NUMBER
]
;
PLAY
IDENTIFIER
^
NUMBER
;
